src/site/content/en/blog/idle-detection/index.md

```diff
@@ -0,0 +1,258 @@
+| 4. Origin trial                          | [In progress][ot]        |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use the Idle Detection API {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Idle Detection API locally, without an origin trial token, enable the
+`#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting with Chrome&nbsp;84, the Idle Detection API will be available as an origin trial.
+The origin trial is expected to end in Chrome 87.
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Feature detection
+
+To check if the Idle Detection API is supported, use:
+
+```javascript
+if ('IdleDetector' in window) {
+  // Idle Detector API supported
+}
+```
+
+### Idle Detection API concepts
+
+The Idle Detection API assumes that there is some level of engagement between the user,
+the user agent (that is, the browser), and the operating system of the device in use.
+This is represented in two dimensions:
+
+- **The user idle state:**
+  `active` or `idle`: the user has or has not
+interacted with the user agent for some period of time.
+- **The screen idle state:**
+  `locked` or `unlocked`: the system has an active screen lock (like a screensaver) preventing
+  interaction with the user agent.
+
+Distinguishing `active` from `idle` requires heuristics that may differ across user, user agent,
+and operating system. It should also be a reasonably coarse threshold
+(see [Security and Permissions](#security-and-permissions)).
+
+The model intentionally does not formally distinguish between interaction with particular content
+(that is, the webpage in a tab using the API), the user agent as a whole, or the operating system;
+this definition is left to the user agent.
+
+### Using the Idle Detection API
+
+The first step when using the Idle Detection API is
+to ensure the `'notifications'` permission is granted.
+The second step is then to instantiate the `IdleDetector`.
+The minimum `threshold` is 60,000 milliseconds (1 minute).
+You can finally start the idle detection by calling the
+`IdleDetector`'s `start()` method.
+It takes an object with the desired idle `threshold` in milliseconds
+and `signal` with an [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+to abort idle detection as parameters.
+
+```js
+const main = async () => {
+  // Feature detection.
+  if (!('IdleDetector' in window)) {
+    return console.log('IdleDetector is not available.');
+  }
+  // Check if notifications permission is granted.
+  const hasPermission = await navigator.permissions.query({name: 'notifications'});
+  if (hasPermission.state !== 'granted') {
+    return console.log('Notifications permission not granted.');
```

Comment:
Should we include an example of requesting notification permission with [`Notification.requestPermission()`](https://developer.mozilla.org/en-US/docs/Web/API/Notification/requestPermission)?

Reply:
Is there something unique about it to this subject? If not, it's better to link to an established article. Otherwise, you'd need to update this article if something changes, assuming somebody remembers it exists.

Reply:
I now link to an article that describes this.